### ISSUE TITLE
Game stops responding after a while

### ISSUE THREAD

**Issue by maggiesz** [Luke100000/minecraft-comes-alive#959]
**Describe the bug**
I have a forge modpack server with my friends that is hosted on my pc. About a week into the server, one person started crashing and then it kind of spread like a plague. In my case, I see it happening a lot especially when running toward my friend's village.  The game just freezes, Java stops responding, and I have to force it to stop. I get the Minecraft -805306369 error, which often corresponds to RAM. All of the players have allocated more RAM to the game and I also allocated more RAM to the server itself. 

Some other stuff I tried:
One of my friends and I also tried reinstalling Java to no help. I have Architectury and the last I think four or five version of it because I saw suggestions that downgrading might work. If there's an obvious fix I missed, sorry for wasting time and thank you in advance for the help. I hope that's not the case though because I spent like hours both searching up stuff from the logs as well as reading through the issue threads on here.

**To Reproduce**
1. Join server/world
2. Run around, most of the time in a pretty specific spot near a village

**Screenshots**
I don't think 

**Version**
MCA Version, Minecraft version, Modloader
Minecraft: 1.19.2
Forge: 43.4.0
MCA: 7.5.14+1.19.2
Architectury: 6.5.69 (I have also tried the latest versions, which didn't seem to affect anything)

**Latest.log**
Append the log, just drag and drop the file here.
You will find the file in the (launcher and profile specific) minecraft save directory, in logs, called `latest.log`. On Windows by default `%appdata%/.minecraft/logs/latest.log`.

[latest.log](https://github.com/user-attachments/files/16115132/latest.log)

**Comment by Luke100000** [comment]
Please update the mod first

**Comment by Luke100000** [comment]
The server is fine, but the clients crash? Can you verify that it is indeed not a OOM by opening F3 and check memory utilization. After a colelction step (when it decreases again) it should never be above 75%. Can you also append crash logs when present?

**Comment by Luke100000** [comment]
My guess here is this OpenGL error: `[01:31:48] [Thread-14/INFO]: OpenGL debug message: id=1282, source=API, type=ERROR, severity=HIGH, message='GL_INVALID_OPERATION error generated. Buffer is unbound or is already unmapped.'`

Try
* Updating graphics drivers
* Removing any optimization mods, client sided

**Comment by maggiesz** [comment]
Thanks for the quick reply! Yeah, only the client crashes. I'm 99% sure it is not OOM. I just tested it and like you described, it never went above 75%. I'll attach a screenshot of the game being frozen. I'm also very skeptical about this being a RAM issue because my friend assigned a ridiculous amount of RAM to the game to test it and the issue didn't go away.

I updated my drivers and removed all optimization mods per your suggestion, but that also didn't help. It's really weird, it happens in different spots, but most often it happens like about 20 blocks away from a village, in a pretty specific spot. 

Another thing that doesn't help is when the game freezes like that and forces the crash, it doesn't create a crash log, only the latest log. It's neither in .minecraft or the Curseforge profile instance. 

I'll go on to testing each mod that I added to this version of the modpack to see if it could be the one causing issues. On an early version of the modpack with a few less mods it wasn't happening, but then again, it took a bit of time for the issue to start happening even after updating the profile with new mods. I'll test it just in case.

**Comment by maggiesz** [comment]
![screen1](https://github.com/Luke100000/minecraft-comes-alive/assets/157187677/06b8471d-cb24-4a36-b372-eea17b5f8445)
(forgot to add the screenshot, my bad)

**Comment by Luke100000** [comment]
Hmm, my last two ideas:

* Change Java version in an attempt to get a different version of the rendering related libs. E.g. some Java 21.
* Change forge version, interestingly this had an effect in the past with similar issues

**Comment by maggiesz** [comment]
I tried out both of your ideas again and nothing changed. However, I was going through all the mods I added like I mentioned earlier and I deleted two files needed for the BetterDogs, BetterCats, and BetterFarmAnimals resourcepacks. The files were called entity_texture_features_forge_1.19.2-6.0.1 and entity_model_features_forge_1.19.2-2.0.2. By now, I should have crashed, but I ran to the village and back many times and the game hasn't crashed, so the cause might have been the cats that spawn in the village. Still, if the issue comes back later it's probably not related to MCA, so I'll be closing the thread. Thanks so much for the suggestions!! I appreciate you taking the time to help.
